templates/f5/0/docker-compose.yml

```diff
@@ -0,0 +1,10 @@
+external-lb:
+  image: rancher/external-lb:v0.1.1
+  command: -debug -provider=f5_BigIP
```

Comment:
@vincent99 I used lowercase f  to match f5 companies name, since it uses lowercase.
also they use uppercase for bigIP product which is their LB. So it 'f5 BIGIP'
I can change to what our standard has been - 
is  -provider=f5BIGIP fine? Or
is  -provider=f5 fine?